### ISSUE TITLE
[UX] Add the ability for default layouts to be cloned (like views).

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#1674]
Simple: a user might not want to start completely from scratch and it might be more convenient for then to clone an existing layout and edit it.

**Comment by docwilmot** [comment]
This already exists: User-created layouts have a 'Clone' option in their dropbuttons. The default layout doesn't but if you add a new layout it uses the blocks and regions and positions from the default layout.

**Comment by klonos** [comment]
Yeah, I am always playing around with the default ones, so I forgot about this. What I mean is that if one customizes the default ones and needs to clone them at that stage, then they should have that option. Is there any reason why we do not allow the default layouts to be cloned?

**Comment by docwilmot** [comment]
Agreed; should we add this to the META?

**Comment by klonos** [comment]
Done :wink:

**Comment by Dinornis** [comment]
👍 Yes, cloning default layouts would be useful.
